e2e/generateDeployments.js

```diff
@@ -74,12 +74,20 @@ async function run() {
 
   const system = deployments.state['provision.system'].artifacts.imports.system;
   const contracts = {};
-
   contracts.CoreProxy = system.contracts.CoreProxy;
   contracts.AccountProxy = system.contracts.AccountProxy;
   contracts.USDProxy = system.contracts.USDProxy;
   contracts.OracleManagerProxy = system.imports.oracle_manager.contracts.Proxy;
 
+  if (deployments.state['provision.legacyMarket']) {
+    contracts.LegacyMarketProxy =
+      deployments.state['provision.legacyMarket'].artifacts.imports.legacyMarket.contracts.Proxy;
+    contracts.V2xSnx =
```

Comment:
Would be good to avoid shortening of names and ideally follow contract name, so in this case it could be 
```suggestion
    contracts.V2xSynthetix =
```